lib/image-diff.js

```diff
@@ -2,16 +2,17 @@
 var fs = require('fs');
 var path = require('path');
 var async = require('async');
-var gm = require('gm').subClass({imageMagick: true});
+var gm;
```

Comment:
We are going to want an instance bound `gm` to prevent sharing a common `gm` instance across different image diffs (e.g. someone might want to run both GraphicsMagick and ImageMagick diffs in the same script)